.npmignore

```diff
@@ -0,0 +1,44 @@
+# Logs
```

Comment:
Honestly, I'm not so sure why there is an .npmignore.
I think that it is generated by the generator in order keep consistency across every definition project: the file is always here, if you want to use it you're free to do so, otherwise just keep it untouched.

I recall that I've read that unless your .gitignore does not match what you want to publish, you should not create an .npmignore. 
Maybe we should open an issue on [the `typings` generator repo](https://github.com/typings/generator-typings) or ask @unional ?